src/components/Notification/Notification.js

```diff
@@ -42,20 +42,17 @@ export class NotificationButton extends Component {
       notificationType,
       ...other
     } = this.props;
-
```

Comment:
Nitpick, but wondering if a tool against blank lines is in use - We tend to keep blank lines wherever it's meaningful.

Reply:
I can add them back, I was just trying to fit more lines on my screen for easier reading. This branch is still WIP so it just contained work I wanted to temporarily save and check into git